rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -536,6 +554,112 @@ void saveOldFederation_after_RSKIP123_should_save_null() {
         );
     }
 
+    private static Stream<Arguments> provideFederationBtcUTXOsTestArguments() {
+        return Stream.of(
+            Arguments.of(false, false, NetworkParameters.ID_TESTNET),
+            Arguments.of(false, false, NetworkParameters.ID_MAINNET),
+            Arguments.of(false, true, NetworkParameters.ID_TESTNET),
+            Arguments.of(false, true, NetworkParameters.ID_MAINNET),
+            Arguments.of(true, false, NetworkParameters.ID_TESTNET),
+            Arguments.of(true, false, NetworkParameters.ID_MAINNET),
+            Arguments.of(true, true, NetworkParameters.ID_TESTNET),
+            Arguments.of(true, true, NetworkParameters.ID_MAINNET)
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideFederationBtcUTXOsTestArguments")
+    void testGetNewFederationBtcUTXOsWithCombinationsOfRSKIPsAndNetworks(boolean isRskip284Active, boolean isRskip293Active, String networkId) {
```

Comment:
```suggestion
    void getNewFederationBtcUTXOs(boolean isRskip284Active, boolean isRskip293Active, String networkId) {
```
`A combination of RSKIPs and Networks` doesn't really say much

Reply:
Updated.